src/cart/test/utest/utest_portnumber.c

```diff
@@ -0,0 +1,186 @@
+/*
+ * (C) Copyright 2020 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+ * The Government's rights to use, modify, reproduce, release, perform, display,
+ * or disclose this software are subject to the terms of the Apache License as
+ * provided in Contract No. 8F-30005.
+ * Any reproduction of computer software, computer software documentation, or
+ * portions thereof marked with this legend must also reproduce the markings.
+ */
+/**
+ * This file is part of CaRT testing.
+ */
+
+/*
+ * This code tests whether a provider returns an error if 2 independent
+ * instances attempts to open the same port number.
+ *
+ * Two child process are forked with the same provider information.
+ * The test is setup so that first child opens the port and then
+ * the second child should fail.
+ * Note, synchronization between child process is performed via sleep
+ * function calls.  The sleep time should prevent any problems, but
+ * beware if an issue arise where the results are swapped.
+ */
+/*
+ * Reference jira ticket DAOS-5732 to include socket and verb tests.
+ */
+#define tests_not_included
```

Comment:
per style for macros/defines we use all uppercase letters

Reply:
Macro now in caps.